legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/executionPlan/executionPlan.pure

```diff
@@ -29,6 +30,7 @@ Class meta::external::store::service::executionPlan::nodes::RestServiceExecution
    params                 : ServiceParameter[*];
    requestBodyDescription : RequestBodyDescription[0..1];
    securitySchemes        : SecurityScheme[*];
```

Comment:
Should we not delete securitySchemes from metamodel now?

Reply:
We cannot delete it. old protocols still using it

Reply:
Should we remove it from VX_X_X though ?